pkgs/top-level/all-packages.nix

```diff
@@ -6776,6 +6776,8 @@ with pkgs;
   code-browser-gtk2 = callPackage ../applications/editors/code-browser { withGtk2 = true; };
   code-browser-gtk = callPackage ../applications/editors/code-browser { withGtk3 = true; };
 
+  CertDump = callPackage ../tools/security/CertDump { };
```

Comment:
According to the contributing guide, package names should not be upper cased or camel cased

Reply:
According to https://nixos.org/manual/nixpkgs/stable/#sec-package-naming:

> The pname attribute should be identical to the upstream package name.

This is the case here?

Reply:
I mean, I would be interested in the line on that because I don't find it in the manual.

Reply:
This line is the most relevant here:

> The pname and the version attribute must not contain uppercase letters — e.g., "mplayer" instead of“MPlayer”`.

Reply:
Right, we should cleanup this in a next PR then.